libunwind/src/Registers.hpp

```diff
@@ -1883,7 +1883,7 @@ inline bool Registers_arm64::validRegister(int regNum) const {
   if (regNum < 0)
     return false;
 #ifdef __CHERI_PURE_CAPABILITY__
-  if ((regNum >= UNW_ARM64_C0) && (regNum <= UNW_ARM64_C31))
+  if (regNum >= UNW_ARM64_C0 && regNum <= UNW_ARM64_C31)
```

Comment:
The brackets are dumb but it's what the local style is... (and setRegister continues to use them)

Reply:
This is just matching Morello.  The x registers still use the extra brackets.  If we'd rather have them I can add them back though.  Will just be a merge conflict downstream.